.github/workflows/release.yml

```diff
@@ -163,6 +140,28 @@ jobs:
       latest_published_version: ${{ needs.build.outputs.RELEASE_VERSION }}
       pre_release: ${{ inputs.pre_release }}
 
+  bump_version:
+    needs: [build, release]
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
Are these permissions not conflicting with each other? I think the `pull-requests: write` permission is enough for that, no? The `contents: write` permission seems to be much broader and allows things we don't need.

References:
https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#pull-requests
https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#contents

Reply:
Not in this case :) 

**`create-pr`** action needs to create a temporary branch to push these staged files (`pyproject.toml`) - hence `contents: write`. Then, creates a PR against `develop` using our temporary branch as the HEAD - hence `pull-requests: write`